app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
- [ ] [OPTIONAL] On subsequent projects that involved implementing devise for signup, I will advise you enable `:confirmable` so that it will allow the email to be confirmed.